libpod/container.go

```diff
@@ -89,6 +89,16 @@ func (ns LinuxNS) String() string {
 	}
 }
 
+// CreateMode is used to define how conmon should start a container
+type CreateMode int
+
+const (
+	// CreateContainer creates a new container
+	CreateContainer CreateMode = iota
+	// RestoreContainer restores an existing container
+	RestoreContainer CreateMode = iota
```

Comment:
Do we see there being more of these in the future? I can't think of obvious ones, so I would lean towards just leaving this a bool argument in OCIRuntime.createContainer